### PR TITLE
better logging when a sync token lookup fails

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?156243

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/ex-submodules/couchforms/models.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from __future__ import absolute_import
 import base64
+from __future__ import absolute_import
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/models.py]: F404 future import(s) ['absolute_import'] after other statements

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by benrudolph** [comment]
not sure what the difference is in the import `couchdbkit` vs `couchdbkit.resources` but both seem valid in the shell

**Comment by czue** [comment]
@benrudolph `couchdbkit` or `couchdbkit.exceptions` are the right places to import from. if anyone ever innocuously deleted this line: https://github.com/dannyroberts/couchdbkit/blob/jsonobject/couchdbkit/resource.py#L30 then these imports would start failing.

**Comment by benrudolph** [comment]
cool thanks

**Pull request merged**